build/config/BUILD.gn

```diff
@@ -113,6 +113,7 @@ config("default_libs") {
       "dnsapi.lib",
       "iphlpapi.lib",
       "msimg32.lib",
+      "ntdll.lib",
```

Comment:
Shouldn't this go here https://github.com/dart-lang/sdk/blob/main/runtime/BUILD.gn#L148, and should it be limited to only some runtime modes?

Reply:
https://github.com/flutter/flutter/issues/124532